runtime/rpc.cpp

```diff
@@ -798,6 +805,7 @@ void process_rpc_error(int32_t request_id, int32_t error_code __attribute__((unu
   }
 
   request->error = error_message;
+  request->error_code = error_code;
```

Comment:
` __attribute__((unused))` at `error_code` param is not needed anymore

Reply:
fix